### PR TITLE
New transformers

### PR DESCRIPTION
**Pull request by asafshen**
Added two transformers:
 - `TimeStampToDate` - Converts UNIX Epoch time to date that is support by demisto 
 - `WhereFieldEquals` - Filter list by certain field (need to done in order to filter nested levels, such as "Email.Headers"

And tests for both..

Also created another artifact - `content_test.zip` - that contains content with all `TestPlaybooks` artifacts 

will also show the second one in the demo next week, I think it can help the SEs...

### CODE REVIEW

File: Scripts/script-WhereFieldEquals.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  var res = value.filter(function(item) {
+     return item && item[field] == equalTo; // using '==' and not '===' to handle number equality better
+  });
+
+  if (getField) {
+      if (typeof getField !== 'string') {
+          logInfo('WhereFieldEquals - got invalid getField [' + getField + ']. Ignoring');
+      } else {
+           res = res.map(function(item) { return item[getField]});
```
Comment (dorsha) [inline Scripts/script-WhereFieldEquals.yml]: make sure res is not nil.. if the above filter returned nothing

File: Scripts/script-WhereFieldEquals.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- name: value
+  required: true
+  default: true
+  description: List to apply the transformer on
+- name: field
+  required: true
+  description: Attribute in the collection's items check equality against 'equalTo'
+- name: equalTo
+  required: true
+  description: Value to filter all items in the collec
```
Comment (dorsha) [inline Scripts/script-WhereFieldEquals.yml]: change collec to collection

File: Scripts/script-WhereFieldEquals.yml (hunk 1 above)
Comment (asafshen) [reply Scripts/script-WhereFieldEquals.yml]: I think `filter` cannot return null/undefined - it always returns an array (perhaps empty one)

**Comment by dorsha** [comment]
@Bars92 you can use the `WhereFieldEquals` transformer (from the next content update).
Or upload it with a different name and id.

**Pull request merged**